src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Properties/Resources.resx

```diff
@@ -146,11 +146,11 @@
     <value>Fail to open file at</value>
   </data>
   <data name="wox_plugin_folder_select_folder_first_result_subtitle" xml:space="preserve">
-    <value>Use &gt; to search within the directory. Use * to search for file extensions. Or use both &gt;*</value>
+    <value>Tip: &gt; to search in folders, * to search file extensions, &gt;* for both.</value>
```

Comment:
This tooltip is hard to understand.  Could we improve the explanation?

Reply:
What do you suggest? It has to be short.

Reply:
@jaimecbernardo
Do you think `Tip: Append ....` is more clear and better?

Reply:
@lncubus , could you please make a suggestion?

Reply:
Okay, how about this one?

`Tip: &gt; to search in folders, * to search file by mask, &gt;* for both.`

Reply:
Sounds good.

Should we use `Tip: Append &gt; to search in folders, * to search file by mask, &gt;* for both.`?